### PR TITLE
Add pre and post install script for symlink

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no 
| BC breaks? | no 
| Deprecations? | no  
| Fixed tickets | fixes # 
| Related issues/PRs | https://github.com/sulu/skeleton/pull/96 Replaced by #5859
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Add pre and post install script for symlink.

#### Why?

Fixes current problems with CKEditor https://github.com/sulu/sulu/issues/5852. Problems with NPM 7: https://github.com/sulu/skeleton/issues/88 and hopefully issues with the windows build also: https://github.com/sulu/skeleton/issues/68

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## 2.1.9
 
+### A new argument `$requestStack` has been added to the `ContentTwigExtension`
+
+Instantiating a `ContentTwigExtension` without the `$requestStack` argument is deprecated.
+
```
Comment (luca-rath) [inline UPGRADE.md]: Why did you add this? It's already [here](https://github.com/sulu/sulu/blob/2.1/UPGRADE.md)

Comment (luca-rath) [reply UPGRADE.md]: Ah, this is not the only wrong thing in here 😂

File: preinstall.js [hunk 2]
```diff
@@ -3,3 +3,5 @@
 if (process.env.npm_execpath.indexOf('npm') === -1) {
     throw new Error('\x1b[31mYou must use "npm install", yarn is not supported\x1b[0m');
 }
+
+require('./symlink.js');
```
Comment (niklasnatter) [inline preinstall.js:10]: think we should only require this if the script is executed inside of the `assets/admin` folder

File: postinstall.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+require('./symlink.js');
```
Comment (niklasnatter) [inline postinstall.js:3]: think we should only require this if the script is executed inside of the `assets/admin` folder

File: webpack.config.js [hunk 4]
```diff
@@ -3,7 +3,6 @@
 /* eslint-disable import/no-dynamic-require */
 const fs = require('fs');
 const path = require('path');
-const babelConfig = JSON.parse(fs.readFileSync(path.resolve(__dirname, '.babelrc'))); // eslint-disable-line no-undef
```
Comment (niklasnatter) [inline webpack.config.js:6]: would use a `babel.config.js` here too

Comment (niklasnatter) [reply webpack.config.js:6]: and move the `postcss.config.cs` into the bundles 🙂

**Comment by alexander-schranz** [comment]
Replaced by #5859
